README.md

```diff
@@ -323,6 +325,25 @@ The Elm library and core packages will be added via Yarn and Elm itself.
 An example `Main.elm` app will also be added to your project in `app/javascript`
 so that you can experiment with Elm right away.
 
+### Coffeescript
+
+To add coffeescript[Coffeescript](http://coffeescript.org/) support,
```

Comment:
coffeescript before [Coffeescript] is unnecessary and CoffeeScript is right, I think.